src/main/steps/applicant/yourdocuments/alldocuments/otherDocuments/content.ts

```diff
@@ -48,16 +48,16 @@ export const form: FormContent = {
 export const generateContent: TranslationFn = content => {
   const translations = languages[content.language]();
   const orders: object[] = [];
-  for (const doc of content.userCase?.otherDocuments || []) {
-    if (doc.value?.documentTypeOther === 'otherReports') {
-      const uid = doc.value.documentOther?.document_url.substring(
-        doc.value.documentOther.document_url.lastIndexOf('/') + 1
+  for (const doc of content.userCase?.citizenUploadedDocumentList || []) {
```

Comment:
Use better looping constructors instead of for to attain performance.

Reply:
Yeah done this with idamId and doc type.